Move ESO to new doc hub (#2716)
giantswarm/docs: src/content/tutorials/continuous-deployment/manage-workload-clusters/index.md

### Setting up secrets {#setting-up-secrets}

> The next step, you configure the keys used by `Flux` in the management cluster to decipher secrets kept in the repository. Our recommendation is to keep secrets encrypted in the repository together with your applications but if your company policy doesn't allow it you can use [`external secret operator`]({{< relref "/tutorials/security/external-secrets-operator/" >}}) to use different sources.

Author: ```suggestion
The next step is to configure the keys used by `Flux` in the management cluster to decipher secrets kept in the repository. We recommend maintaining secrets encrypted in the repository alongside your applications. But if your company policy doesn't allow it, you can use [`external secret operator`]({{< relref "/tutorials/security/external-secrets-operator/" >}}) to use different sources.
```
